mmv1/products/networksecurity/SecurityProfileGroup.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'SecurityProfileGroup'
+base_url: '{{parent}}/locations/{{location}}securityProfileGroups'
+create_url: '{{parent}}/locations/{{location}}/securityProfileGroups?securityProfileGroupId={{name}}'
+self_link: '{{parent}}/locations/{{location}}/securityProfileGroups/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  A security profile group defines a container for security profiles.
+references:
+  !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/firewall/docs/reference/network-security/rest/v1beta1/organizations.locations.securityProfileGroups'
+  guides:
+    'Security profile groups overview': 'https://cloud.google.com/firewall/docs/about-security-profile-groups'
+    'Create and manage security profile groups': 'https://cloud.google.com/firewall/docs/configure-security-profile-groups'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+  include_project: true
+import_format: ['{{%parent}}/locations/{{location}}/securityProfileGroups/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_security_security_profile_group_basic'
+    primary_resource_id: 'default'
+    min_version: beta
+    test_env_vars:
+      org_id: :ORG_ID
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the security profile group resource.
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    default_value: 'global'
+    description: |
+      The location of the security profile group.
+      The default value is `global`.
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'parent'
+    description: |
+      The name of the parent this security profile group belongs to.
+      Format: organizations/{organization_id}.
```

Comment:
According to the docs, this can also be `projects/` - is that accurate? If so, we often split resources that can exist at different levels into level-specific resources (because in practice they can end up behaving fairly differently.)

Reply:
I checked and this resource can only be defined under the organization. I don't think the team has plans to move it to the project.